src/deploy-tes-on-azure/Program.cs

```diff
@@ -32,7 +34,13 @@ private static async Task InitializeAndDeployAsync(string[] args)
                 PrintWelcomeScreen();
             }
 
-            Environment.Exit(await new Deployer(configuration).DeployAsync());
+            if (!AzureEnvironmentExtension.IsAvailableEnvironmentName(configuration.AzureName))
```

Comment:
this should all be moved to ValidateInitialCommandLineArgsAsync

Reply:
Already moved to. 
Added embedded function in ValidateInitialCommandLineArgsAsync:
```csharp
            void ThrowIfAzureEnvironmentNotExists(string azureEnvironmentName)
            {
                if (!AzureEnvironmentExtension.IsEnvironmentNameAvailable(azureEnvironmentName))
                {
                    throw new ValidationException($"Specified cloud {azureEnvironmentName} does not exist.");
                }
            }
```